assets/src/less/components/comments/layout.less

```diff
@@ -0,0 +1,42 @@
+.comments {
+	.stack--loose;
+	padding-inline-start: @space-vw-2;
+	padding-inline-end: @space-vw-2;
+}
+
+.comments__content {
+	.clearfix--after;
+	.stack;
+	.margin-inline( auto );
+	max-inline-size: 64rem;
+	> * {
+		max-inline-size: 32rem;
+	}
+
+	> .alignleft {
+		margin-inline-start: 0;
+		margin-inline-end: auto;
+	}
+
+	> .aligncenter {
+		.margin-inline( auto );
+	}
+
+	> .alignright {
+		margin-inline-start: auto;
+		margin-inline-end: 0;
+	}
+
+	> .alignwide {
+		max-inline-size: 100%;
+	}
+
+	> .alignfull {
+		max-inline-size: none;
+		.margin-inline( -@space-vw-2 );
+	}
+
+	> .woocommerce {
+		max-inline-size: 100%;
+	}
```

Comment:
I don't think we'll ever use these inside of the comments content. I should have been more clear in the issue. Sorry about that.

Reply:
Np, cleared out.